src/codegen/templates/library_interpreter/default_c_function_call.py.mako

```diff
@@ -7,18 +7,24 @@
     )
     from codegen.utilities.text_wrappers import wrap, docstring_wrap
 
-    function_call_args = generate_interpreter_function_call_args(function)
+    function_call_args, dateTime_args = generate_interpreter_function_call_args(function)
 
     # samps_per_chan_param includes the keyword argument (samps_per_chan_read=
     # or samps_per_chan_written=)
     samps_per_chan_param = get_samps_per_chan_read_or_write_param(function.base_parameters)
 %>\
         cfunc = lib_importer.${'windll' if function.calling_convention == 'StdCall' else 'cdll'}.DAQmx${function.c_function_name}
+<%
+    argtypes = get_argument_types(function)
+%>\
+%if dateTime_args:
+        ${'\n\t\t'.join(dateTime_args)}
```

Comment:
Please don't wrap lines this way.

1. Using \n in a mako template is a red flag. Mako preserves \r\n vs. \n when reading and rendering templates, so many scripts open the output file as binary to avoid newline conversion.
2. This code generator doesn't use tabs, does it?
3. This code generator has a wrap() function for wrapping argument lists.
4. This code generator uses %for for iterating over lines of code.

Also this is moot because I think you should delete dateTime_args.